contracts/rounds/AdjustableRoundsManager.sol

```diff
@@ -26,7 +26,7 @@ contract AdjustableRoundsManager is RoundsManager {
     }
 
     function blockHash(uint256 _block) public view returns (bytes32) {
-        require(_block <= blockNum() - 256);
```

Comment:
This was a bug (though it only affected tests and not a live deployed contract system because in production `AdjustableRoundsManager` would never be used - only `RoundsManager`). This change makes sure that `blockHash` reverts if the block number provided is not greater than the current block - 256 to match up with the behavior of `blockHash` in `RoundsManager`.